### ISSUE TITLE
EAC3 audio interpreted as part of release group

### ISSUE THREAD

**Issue by hpsbranco** [guessit-io/guessit#431]
```python
>>> guess = guessit('The.Expanse.S02E08.720p.WEBRip.x264.EAC3-KiNGS.mkv')
>>> 'audio_codec' in guess
False
>>> guess['release_group']
'EAC3-KiNGS'
```
May I submit a pull request fixing this?

**Comment by ratoaq2** [comment]
Sure, any contribution is welcome.

You just need to have the tests green

**Comment by ratoaq2** [comment]
Fixed by #445
